README.md

```diff
@@ -1,70 +1,54 @@
-# Getting Started with Create React App
+# Weather App - Capstone Project
```

Comment:
-[ ] In your README, kindly add an **Acknowledgements** section or similar that gives appropriate credit to Nelson Sakwa, the author of the original design, as required by the Creative Commons License.

Reply:
Thank! I can't Believe I forgot to do it

Reply:
Well resolved this!